src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -442,49 +479,49 @@ public function withCount($relations)
     /**
      * Add subselect queries to include the max of the relation's column.
      *
-     * @param  string  $relation
+     * @param  string|array  $relations
      * @param  string  $column
      * @return $this
      */
-    public function withMax($relation, $column)
+    public function withMax($relations, $column = null)
```

Comment:
This is a breaking change to the method signature. All of these method signature changes below are breaking changes as well.

Reply:
These methods did support arrays as the first parameter before, too. It was just not documented. But you might be more concerned about the second parameter becoming optional. As I mentioned in the comment above, nothing should break if it keeps using the former way implemented in #34965. But maybe I have a wrong understanding of what a breaking change.

Reply:
https://3v4l.org/VNAms